src/envo/envo-edit.owl

```diff
@@ -41989,8 +42006,147 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501354> <http://purl.obolibrar
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Private_prison>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501355> "A prison facility which is owned by the state and operated by a private corporation."@en)
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03501355> <http://orcid.org/0000-0002-8844-9165>)
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03501355> "2021-05-19T17:30:47Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501355> "2021-12-09T00:00:00Z"^^xsd:dateTime)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501355> "public-private prison"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501355> <http://purl.obolibrary.org/obo/ENVO_03501204>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501355> <http://purl.obolibrary.org/obo/RO_0002577>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501355> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_01000610>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03501356> (conveyor system)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> < https://www.conveyorsystemsltd.co.uk/news/168/126/Roller-Conveyor-vs-Belt-Conveyor.htm>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://vocab.getty.edu/page/aat/300024559 >) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501356> "A system which is composed of one or more machines that can continuously transport material from one location to at least one other location."@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03501356> <https://orcid.org/0000-0002-2738-824X>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03501356> "2021-12-09T00:00:00Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasBroadSynonym> <http://purl.obolibrary.org/obo/ENVO_03501356> "conveyor"@en)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasNarrowSynonym> <http://purl.obolibrary.org/obo/ENVO_03501356> "connveyor belt"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03501356> "conveyor system"^^xsd:string)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501356> <http://purl.obolibrary.org/obo/ENVO_00000010>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501356> <http://purl.obolibrary.org/obo/RO_0002577>)
```

Comment:
on reflection this RO class was a mistake. I think the dual is-a assertion is a bit of a red flag

however, ok to merge this and we can address the whole 'system' issue separately